data/transit/route/train.json

```
@@ -476,6 +476,18 @@
         "route": "train"
       }
     },
+    {
+      "displayName": "Metro-North Railroad",
+      "id": "mnr-44bbb3",
+      "locationSet": {"include": ["us"]},
+      "tags": {
+        "network": "Metro-North Railroad",
+        "network:wikidata": "Q125908",
+        "network:wikipedia": "en:Metro-North Railroad",
+        "operator": "Metro-North Railroad",
```

I know it's kind of redundant, but if adding `operator` we should add the wikipedia and wikidata for those also.  (I'll update the docs to suggest this also).

```suggestion
        "operator": "Metro-North Railroad",
        "operator:wikidata": "Q125908",
        "operator:wikipedia": "en:Metro-North Railroad",
```